public/modules/loggedIn/users/directive_simple_user.html

```diff
@@ -0,0 +1,8 @@
+<!-- example usage: <oms-simple-user userid="{{vm.recipe.created_by}}"></oms-simple-user> -->
+
+<span ng-show="!message">
+{{ fetched_user.first_name }} {{ fetched_user.last_name }}
+</span>
+<span ng-show="message">
+{{ message }}
+</span>
```

Comment:
I keep seeing these white-space 'issues' in your commits. While I do not find them game-breaking, maybe it would be wise to have a look at your editor settings or something?
(this also prevents my editor automatically correcting them whenever I work from something you made :))

Reply:
okay, maybe we could commit a .editorconfig in oms-docker and I can set up my sublime to use it?

Reply:
To be honest I don't really know how to do that works (never really had the need to do such a thing).
But it sounds good? 👍 
Should we make a JIRA issue about this?

Reply:
Yeah, I think @serge1peshcoff has more experience in this!